### PR TITLE
Milestone 2-4 retrospective update

### PR DESCRIPTION
**Pull request by Wagoory**
worked the the retrospectives for milestone 2 and 4

### CODE REVIEW

File: 4_communicating_results/retrospective_of_4_communicating_results.md [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+3- We worked on describing and identifing our target audience
+4- We worked on the script for our final presentation
+5- We worked on finalizing the ChatGPT-simulation analysis
 
 ## Challenges Faced
- 
-
-## Tools and Technique Evaluation
- 
+NA
```
Comment (MaabTaha) [inline 4_communicating_results/retrospective_of_4_communicating_results.md]: Coming up with the artifact was a big challenge that required a lot of brainstorming.
We faced problems in finding a time where enough people in the team was available to meet.

File: 4_communicating_results/retrospective_of_4_communicating_results.md [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
 ## Team Dynamics and Collaboration
-
-
-## Start and Stop doing
+Some team members where not available, but still the work flow has been good and the rest of the team is participating perfectly.
+We have two meetings per week, which through we discuss what needs to be done through out the milestone and the team divides the work between them
 
 ## Conclusions
+At the end of this project. We think it was a great experience and a lot of new tools and communication ways where developed to each of the members
+
```
Comment (MaabTaha) [inline 4_communicating_results/retrospective_of_4_communicating_results.md:23]: Thinking through the eyes of a data scientist in this stage caused us to be stuck for a long time. One of the major things that helped us was to put ourselves in the shoes of our audience and to speak as if we were speaking to our families. We had to take it less seriously and have more fun with it.

File: 4_communicating_results/retrospective_of_4_communicating_results.md (hunk 1 above)
Comment (Wagoory) [reply 4_communicating_results/retrospective_of_4_communicating_results.md]: Fixed the changes

File: 4_communicating_results/retrospective_of_4_communicating_results.md (hunk 2 above)
Comment (Wagoory) [reply 4_communicating_results/retrospective_of_4_communicating_results.md:23]: Done

**Pull request merged**